src/app/screens/LNURLWithdraw/index.test.tsx

```diff
@@ -26,6 +30,16 @@ const mockDetails: LNURLWithdrawServiceResponse = {
   domain: "lnurl.fiatjaf.com",
 };
 
+const mockDetailsLnBits: LNURLWithdrawServiceResponse = {
+  tag: "withdrawRequest",
+  callback:
+    "https://legend.lnbits.com/withdraw/api/v1/lnurl/cb/D7paixQdqcsm3VJrstczsQ?id_unique_hash=a5BgmGCDZfz2eMQCofxquP",
```

Comment:
I'd prefer to use real mock values and data that we control
also here it is not obvious from the URL that this link does not work.

Reply:
> I'd prefer to use real mock values and data that we control 

Please elaborate, not sure what you mean with "real mock data" and data we control. What should we use for example?

> Also here it is not obvious from the URL that this link does not work.

This is just a test for the UI. Not sure how this is relevant here.

Reply:
why not use "https://example.com/invalid-link"

when I read that code it looks like a very real link and we do not know what happens behind it.